api/goldens/aptos_api__tests__transactions_test__test_get_txn_execute_failed_by_invalid_script_function_arguments.json

```diff
@@ -6,7 +6,7 @@
   "event_root_hash": "",
   "gas_used": "1",
   "success": false,
-  "vm_status": "Transaction Executed and Committed with Error FAILED_TO_DESERIALIZE_ARGUMENT",
+  "vm_status": "Transaction Executed and Committed with Error FUNCTION_RESOLUTION_FAILURE",
```

Comment:
this looks changed too

Reply:
Nice catch thanks! I missed a couple of TestCoin references.